style.css

```diff
@@ -156,20 +421,176 @@ html {
   width: 30px;
 }
 
-/* Media Queries */
+.card-container {
```

Comment:
- [ ] I noticed that the main container of the project does not have the required "display:grid" property as specified in the project requirements.
In order to maintain consistency and ensure that the project meets the necessary standards, it's important that this requirement is followed. Please make the necessary changes to the code and resubmit your pull request.

<img width="740" alt="image" src="https://user-images.githubusercontent.com/88894525/236070207-53bf7ace-91ab-4168-bd07-28a3c459682a.png">